.github/workflows/blackhole-post-commit.yaml

```diff
@@ -29,6 +29,7 @@ jobs:
     uses: ./.github/workflows/build-artifact.yaml
     secrets: inherit
     with:
+      os: "ubuntu-20.04-amd64"
```

Comment:
Ugh this seems like something gross in `build-artifact` you had to deal with. It's a little jarring to see `ubuntu-20.04` and `ubuntu-20.04-amd64` both used as an `os:` parameter value (even if it's for different workflows).

Now, given the size of our team and heavily we rely on socialization anyway for information about our workflows, I don't really care what this is. I'm just wondering if you think this is an opportunity to clean up the values we're providing here.

Reply:
`docker_os_arch` seems like a more appropriate name for something like this, as you set in another file in this PR

Reply:
If we're going to spend time cleaning up, then I would want 2 separate params.  OS and arch are othogonal.  Baking them into a single param conflates matters.

Right now we're only dealing with x86_64, so we can assume that and only add an arch later when we expand to that.  Happy to either clean it up here or as a follow-up.

Reply:
No big deal, made a P3 issue for later https://github.com/tenstorrent/tt-metal/issues/15611